scripts/minikube.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash -e
-
+# shellcheck disable=SC1091
```

Comment:
please only do this where needed, not for the whole script

Reply:
Done.